cellprofiler/modules/graphpartition.py

```diff
@@ -0,0 +1,99 @@
+"""
+
+Graph partition
+
+"""
+
+import cellprofiler.image
+import cellprofiler.module
+import cellprofiler.object
+import cellprofiler.setting
+import skimage.measure
+import skimage.segmentation
+
+
+class GraphPartition(cellprofiler.module.Module):
+    category = "Image segmentation"
+    module_name = "Graph partition"
```

Comment:
We need consistency between camel case and whatever this format is called (regular?).